lists/bethesda.yaml

```diff
@@ -0,0 +1,18 @@
+-
+    domain: 'schaken-mods.com'
+    advertising: 5
+    redistribution: 4
+    miscellaneous: 5
+    notes: 'also redistributes child nudity mods'
+-
+    domain: 'http://www.9dmdamaomod.com'
+    advertising: 1
+    redistribution: 4
+    miscellaneous: 4
+    notes: 'chinese site that rehosts quite a bit of english content. generally safe, however does require baidu account'
+-
+    domain: 'allthefallen.moe'
+    advertising: 1
+    redistribution: 4
+    miscellaneous: 5
+    notes: 'specifically designed for child nudity mods. includes reposts of others works, but with nude children added'
```

Comment:
This needs to be reworked a bit to comply with the [new api format](https://github.com/StopModReposts/Illegal-Mod-Sites/wiki/API-access-and-formats). Mainly, remove ratings and replace them with `notes` and `reason`